ui/app/styles/components/masked-input.scss

```diff
@@ -70,6 +70,7 @@
   .button.masked-input-toggle,
   .button.copy-button,
```

Comment:
Will remove the `copy-button` class from everywhere in this file once the final copy button using it (in hover-copy-button.hbs) gets phased out